app/javascript/packages/components/status-page.tsx

```diff
@@ -86,7 +86,7 @@ function StatusPage({
 
   return (
     <>
-      <img src={src} width={54} height={54} alt={alt} className="display-block margin-bottom-4" />
+      <img src={src} alt={alt} className="display-block margin-bottom-4 lg-icon" />
```

Comment:
It didn't occur to me 'til now, but the consolidation of attributes to the stylesheet does have one  major disadvantage: It's good practice to _always_ assign `width` and `height` to an `<img />` tag so that the browser can render the layout as if the image were there -- before network resources such as the image itself or associated stylesheets -- in order to avoid page layout shifts when those resources do eventually load.

- https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/
- https://www.aleksandrhovhannisyan.com/blog/setting-width-and-height-on-images/

Reply:
Excellent point. I'm thinking now maybe a custom tag helper would make this all just a bit simpler in that case.

Reply:
Maybe a ViewComponent / React component could solve all of our outstanding needs?

Component: `AlertIcon` renders the image
Styles: `.alert-icon` self-contained to `app/assets/stylesheets/components/_alert-icon.scss`
Locale strings: Located in `components.alert_icon.x`